project/Dependencies.scala

```diff
@@ -11,7 +11,8 @@ object Dependencies {
 
   val AkkaVersion = "2.5.4"
   val AkkaHttpVersion = "10.0.9"
-  val ScalaVersion = "2.11.11"
+  val ScalaVersions = Seq("2.11.11", "2.12.3")
```

Comment:
I'm swapping the versions. On a previous commit, Scala 2.12 was the default one. 

The rationale behind this swap (eg: 2.11 as default) is that maven plugin only support (for the moment) one Scala version. If we make Scala 2.12 the default one we will force potential users to move to Scala 2.12. The chance is minimal, but we may have users using maven and having it configured to compile Scala.